questions/48/question.cpp

```diff
@@ -5,12 +5,12 @@
 int main() {
   std::string x = "x";
 
-  std::async(std::launch::async, [&x]() {
+  (void)std::async(std::launch::async, [&x]() {

```

Comment:
I think we don't need to suppress the warnings. They help users to learn more about the question.

Reply:
I also prefer to avoid the cast to `void` since I think it complicates the question.